<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRmKCM</class>
 <widget class="QWidget" name="AddRmKCM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>468</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Add and Remove Software</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="3" column="0" colspan="2">
    <widget class="KpkTransactionBar" name="transactionBar" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="browse">
      <attribute name="title">
       <string>Browse</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item row="2" column="0">
        <widget class="KpkCategorizedView" name="homeView">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideNone</enum>
         </property>
         <property name="viewMode">
          <enum>QListView::IconMode</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="backTB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Back</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KLineEdit" name="searchKLE">
           <property name="clickMessage">
            <string>Search packages</string>
           </property>
           <property name="showClearButton" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="margin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="filtersTB">
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="text">
            <string>Filters</string>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <widget class="KpkSearchableTreeView" name="packageView">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QWidget" name="stackedWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Installed Packages</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="1" column="0" colspan="3">
        <widget class="KpkSearchableTreeView" name="installedView">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="exportInstalledPB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Export Installed Package List...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="importInstalledPB">
         <property name="text">
          <string>Install Packages from List...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>No Change Pending</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="KpkCategorizedView" name="changesView">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionFindName">
   <property name="text">
    <string>Find by &amp;name</string>
   </property>
  </action>
  <action name="actionFindDescription">
   <property name="text">
    <string>Find by &amp;description</string>
   </property>
  </action>
  <action name="actionFindFile">
   <property name="text">
    <string>Find by f&amp;ile name</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KpkTransactionBar</class>
   <extends>QWidget</extends>
   <header location="global">KpkTransactionBar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KpkCategorizedView</class>
   <extends>QListView</extends>
   <header>KpkCategorizedView.h</header>
  </customwidget>
  <customwidget>
   <class>KpkSearchableTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">KpkSearchableTreeView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>searchKLE</tabstop>
  <tabstop>filtersTB</tabstop>
  <tabstop>homeView</tabstop>
  <tabstop>packageView</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>backTB</tabstop>
  <tabstop>installedView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
